pkg/controller/direct/compute/forwardingrule_controller.go

```diff
@@ -358,6 +362,55 @@ func (a *forwardingRuleAdapter) Create(ctx context.Context, createOp *directbase
 		return fmt.Errorf("getting ComputeForwardingRule %q: %w", a.fullyQualifiedName(), err)
 	}
```

Comment:
Per our discussion, I think we want to update the forwarding rule status here, right?

Reply:
I think that's blocked on my getting my status changes merged though - I'll work on that!